packages/cyberstorm/src/components/Header/Header.tsx

```diff
@@ -64,65 +66,69 @@ export function Header() {
+          <RadixDropDown.Item>
+            <div className={styles.dropDownUserInfo}>
+              {userData.imageSource ? (
+                <Avatar src={userData.imageSource} />
+              ) : null}
+              <div className={styles.dropdownUserInfoDetails}>
+                <div className={styles.dropdownUserInfoDetails_userName}>
+                  {userData.name}
+                </div>
+                <div className={styles.dropdownUserInfoDetails_description}>
+                  My profile
+                </div>
+              </div>
             </div>
-          </div>
-        </div>
-      </RadixDropDown.Item>
-    </UserLink>,
-    <DropDownDivider key="2" />,
-    <TeamsLink key="3">
-      <DropDownItem
-        content={
-          <DropDownLink
-            leftIcon={<FontAwesomeIcon icon={faUsers} fixedWidth />}
-            label="Teams"
+          </RadixDropDown.Item>
+        </UserLink>,
+        <DropDownDivider key="2" />,
+        <TeamsLink key="3">
+          <DropDownItem
+            content={
+              <DropDownLink
+                leftIcon={<FontAwesomeIcon icon={faUsers} fixedWidth />}
+                label="Teams"
+              />
+            }
           />
-        }
-      />
-    </TeamsLink>,
-    <a href="/subscriptons" key="4">
-      <DropDownItem
-        content={
-          <DropDownLink
-            leftIcon={<FontAwesomeIcon icon={faCreditCard} fixedWidth />}
-            label="Subscriptions"
+        </TeamsLink>,
+        <a href="/subscriptons" key="4">
+          <DropDownItem
+            content={
+              <DropDownLink
+                leftIcon={<FontAwesomeIcon icon={faCreditCard} fixedWidth />}
+                label="Subscriptions"
+              />
+            }
           />
-        }
-      />
-    </a>,
-    <SettingsLink key="5">
-      <DropDownItem
-        content={
-          <DropDownLink
-            leftIcon={<FontAwesomeIcon icon={faCog} fixedWidth />}
-            label="Settings"
+        </a>,
+        <SettingsLink key="5">
+          <DropDownItem
+            content={
+              <DropDownLink
+                leftIcon={<FontAwesomeIcon icon={faCog} fixedWidth />}
+                label="Settings"
+              />
+            }
           />
-        }
-      />
-    </SettingsLink>,
-    <DropDownDivider key="6" />,
-    <a href="/logout" key="7">
-      <DropDownItem
-        content={
-          <DropDownLink
-            leftIcon={<FontAwesomeIcon icon={faSignOut} fixedWidth />}
-            label="Log Out"
+        </SettingsLink>,
+        <DropDownDivider key="6" />,
+        <a href="/logout" key="7">
+          <DropDownItem
+            content={
+              <DropDownLink
+                leftIcon={<FontAwesomeIcon icon={faSignOut} fixedWidth />}
+                label="Log Out"
+              />
+            }
           />
-        }
-      />
-    </a>,
-  ];
+        </a>,
+      ]
+    : [];
```

Comment:
This should be turned into a standalone component or function instead of using a massive multi-line ternary operator.